frontend/src/services/volunteer/volunteer.api.ts

```diff
@@ -48,17 +48,11 @@ export const updateVolunteer = async (
   id: string,
   data: VolunteerFormTypes,
 ): Promise<IVolunteer> => {
-  console.log({
-    ...data,
-    branch: data.branch.value,
-    role: data.role.value,
-    department: data.department.value,
-  });
   return API.patch(`volunteer/${id}`, {
     ...data,
-    branch: data.branch.value,
-    role: data.role.value,
-    department: data.department.value,
+    branchId: data.branch?.key,
```

Comment:
Technical task: patch for update (send only changes)

Reply:
Done